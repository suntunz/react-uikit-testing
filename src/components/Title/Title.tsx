import React from 'react'

export interface ITitleProps {
    text: string
}

const Title = ({ text }: ITitleProps) => {
    return (
        <h1 className='text-[36px] text-gray-600 md:text-purple-800 lg:text-yellow-700 xl:text-bitkub-600 2xl:text-pink-700'>
            {text}
        </h1>
    )
}

export default Title
