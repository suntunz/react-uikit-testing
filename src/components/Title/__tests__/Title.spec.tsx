import React from 'react'
import { create } from 'react-test-renderer'

import Title from '../Title'

describe('Title component', () => {
    it('should match previous snapshot', () => {
        const component = create(<Title text='Test' />)
        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})
