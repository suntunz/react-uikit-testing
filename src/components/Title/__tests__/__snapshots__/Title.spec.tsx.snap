// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component should match previous snapshot 1`] = `
<h1
  className="text-[36px] text-gray-600 md:text-purple-800 lg:text-yellow-700 xl:text-bitkub-600 2xl:text-pink-700"
>
  Test
</h1>
`;
